src/views/forms/FormSubWalletCreation/FormSubWalletCreationTs.ts

```diff
@@ -190,19 +194,41 @@ export class FormSubWalletCreationTs extends Vue {
    * Submit action asks for account unlock
    * @return {void}
    */
+  public currentWallet: WalletsModel
+  ledgerForm = formDataConfig.ledgerImportForm
+  name: string | undefined
+  address: string | undefined
+  publicKey: string | undefined
+  active: boolean | undefined
+  path: string
+  sourceType: Number
```

Comment:
please document the need for each of those properties + consistently descript them in code.